lms/static/certificates/sass/_layouts.scss

```diff
@@ -218,15 +225,11 @@
         .organization {
             @include size(100%);
             position: relative;
-            margin-bottom: spacing-vertical(small);
+
         }
 
         .organization-logo {
-            max-width: 80%;
```

Comment:
NOTE: simplifying layout rules for org logos so that they play better with print styling.